site/ohs_configuration.md

```diff
@@ -0,0 +1,49 @@
+## Configuring Oracle HTTP Server (OHS)
+
+Oracle HTTP Server (OHS) can be configured and updated with Create Domain, Update Domain, and Deploy Applications Tools, starting with WDT release 1.8.0. OHS configuration is only supported for offline mode in WDT. The OHS configuration can be discovered using the Discover Domain Tool, in offline mode only.
+ 
+### Prerequisites
+
+In order to configure and use OHS, the product must be installed in the Oracle Home used to create the domain. You can download OHS [here](https://www.oracle.com/middleware/technologies/webtier-downloads.html).
+
+The OHS template must be present in the WDT domain type definition file used to create or update the domain. See [Domain Type Definitions](type_def.md) for more information on creating a custom definition.
```

Comment:
For more information on creating a custom definition, see [Domain Type Definitions](type_def.md).